docs/content/configuration.md

```diff
@@ -548,7 +548,7 @@ The following signing algorithms are supported:
 
 ### Caching
 
-Caching represents the configuration of the inter-query cache that built-in functions can utilize.
+Caching represents the configuration of the inter-query cache that built-in functions can utilize. Be aware that the total memory used by the cached items is significantly more than the bytes tracked for the max limit, around 20-30 times more, depending on the size of the data. For example, a `maz_size_bytes` of `750000` might result in OPA using around `18750000` bytes of memory space. You can use `pprof` to monitor usage for your particular use case.
```

Comment:
Let's file an issue for this and get that resolved asap. We can remove this comment or add a note in the docs.